crates/bevy_core/src/lib.rs

```diff
@@ -57,7 +57,9 @@ fn register_rust_types(app: &mut App) {
         .register_type::<HashSet<String>>()
         .register_type::<Option<String>>()
         .register_type::<Option<bool>>()
+        .register_type::<Option<f32>>()
```

Comment:
Non-blocking: Should we also register other `Option` primitives? Like `Option<usize>`, etc.?